pkgs/tools/misc/anystyle-cli/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildRubyGem
+, bundlerEnv
+, ruby
+, poppler_utils
+}:
+let
+  deps = bundlerEnv rec {
+    name = "anystyle-cli-${version}";
+    source.sha256 = lib.fakeSha256;
+    version = "1.3.1";
+    inherit ruby;
+    gemdir = ./.;
+    gemset = lib.recursiveUpdate (import ./gemset.nix) {
+      anystyle.source = {
+        remotes = ["https://rubygems.org"];
+        sha256 = "1w79zcia60nnnyrmyvpd10pmxrpk5c7lj9gmmblhwi8x5mfq9k0n";
+        type = "gem";
+      };
+    };
+  };
+in
+buildRubyGem rec {
+
+  inherit ruby;
+  name = "${gemName}-${version}";
```

Comment:
I'm not experienced at all with `buildRubyGem`, is it possible this is redundent? Or at least could be replaced with `pname = gemName;`?

Reply:
You're right, fixed.